pkg/controller/registry/resolver/cache/cache.go

```diff
@@ -139,7 +139,7 @@ func (c *NamespacedOperatorCache) Error() error {
 		err := snapshot.err
 		snapshot.m.RUnlock()
 		if err != nil {
-			errs = append(errs, fmt.Errorf("failed to populate resolver cache from source %v: %w", key.String(), err))
+			errs = append(errs, fmt.Errorf("error using catalog %s (in namespace %s): %w", key.Name, key.Namespace, err))
```

Comment:
This error is kinda word, would it make more sense to be the following?
```suggestion
			errs = append(errs, fmt.Errorf("error using catalogsource %s/%s: %w", key.Namespace, key.Name, err))
```
i.e. refer to the actual resource type (catalogsource) and also use namespace/name format

Reply:
Noting that my suggestion was used below for another error you added.

Reply:
That's fair, updated the PR with your suggestion.